Makefile

```
@@ -38,19 +63,48 @@ test: build-python ## Run tests with warnings treated as errors
+
+ensure-whitaker-tools:
+	$(CARGO) dylint --version >/dev/null
+
+$(WHITAKER_SRC)/.git:
+	mkdir -p "$(WHITAKER_ROOT)"
+	if [ ! -d "$(WHITAKER_SRC)/.git" ]; then \
+		git clone --depth 1 --branch "$(WHITAKER_TAG)" \
+			"$(WHITAKER_REPO)" "$(WHITAKER_SRC)"; \
+	fi
+
+$(WHITAKER_LIBRARY): ensure-whitaker-tools $(WHITAKER_SRC)/.git
```

checkmake (0.3.2)

[warning] Target body for "$(WHITAKER_LIBRARY)" exceeds allowed length of 5 lines (10).

(maxbodylength)